<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings::CSettingsWidget</class>
 <widget class="QWidget" name="Settings::CSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>26</width>
    <height>16</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="CParametersWidget" name="parametersPanel" native="true"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="settingsStack">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="Settings::CLanguageWidget" name="pageLanguage"/>
     <widget class="Settings::CRegionalSettingsWidget" name="pageRegionalSettings"/>
     <widget class="Settings::CDateTimeWidget" name="pageDateTime"/>
     <widget class="Settings::CAlarmSettingsWidget" name="pageAlarm"/>
     <widget class="Settings::COvenSettingsWidget" name="pageOven"/>
     <widget class="CAgitationSettingsWidget" name="pageAgitation"/>
     <widget class="CDataManagementWidget" name="pageDataManagement"/>
     <widget class="CServiceSettingsWidget" name="pageService"/>
     <widget class="CFileViewWidget" name="pageEventView"/>
     <widget class="Settings::CNetworkSettingsWidget" name="pageNetwork"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Settings::CLanguageWidget</class>
   <extends>QWidget</extends>
   <header>Settings/Include/LanguageWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Settings::CRegionalSettingsWidget</class>
   <extends>QWidget</extends>
   <header>Settings/Include/RegionalSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Settings::CDateTimeWidget</class>
   <extends>QWidget</extends>
   <header>Settings/Include/DateTimeWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Settings::CAlarmSettingsWidget</class>
   <extends>QWidget</extends>
   <header>Settings/Include/AlarmSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Settings::COvenSettingsWidget</class>
   <extends>QWidget</extends>
   <header>Settings/Include/OvenSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CAgitationSettingsWidget</class>
   <extends>QWidget</extends>
   <header>Settings/Include/AgitationSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CDataManagementWidget</class>
   <extends>QWidget</extends>
   <header>Settings/Include/DataManagementWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CServiceSettingsWidget</class>
   <extends>QWidget</extends>
   <header>Settings/Include/ServiceSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CParametersWidget</class>
   <extends>QWidget</extends>
   <header>Settings/Include/ParametersWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Settings::CNetworkSettingsWidget</class>
   <extends>QWidget</extends>
   <header>Settings/Include/NetworkSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CFileViewWidget</class>
   <extends>QWidget</extends>
   <header>Settings/Include/FileViewWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
